### PR TITLE
Issue 5436 - WW Influent CBOD5 help text

### PR DESCRIPTION
**Pull request by ysintertech**
connects #5436

### CODE REVIEW

**Comment by nbintertech** [comment]
Github isn't allowing file upload right now, msg me if you need reference...

The validation error border style needs some work and the result should show a dash if it has no value. The tab button style has been changed so it's completely white when active - that may have been introduced elsewhere, but check it out please.

**Comment by nbintertech** [comment]
Github isn't allowing file upload right now, msg me if you need reference...

The validation error border style needs some work and the result should show a dash if it has no value. The tab button style has been changed so it's completely white when active - that may have been introduced elsewhere, but check it out please.

File: src/app/shared/models/waste-water.ts [hunk 1]
```diff
@@ -41,6 +41,9 @@ export interface WasteWaterData {
 export interface ActivatedSludgeData {
     Temperature: number,
     So: number,
+    isUserDefinedSo: boolean,
```
Comment (nbintertech) [inline src/app/shared/models/waste-water.ts:44]: Test these fields with an old assessment, and update update-data.service if necessary. isUserDefinedSo may come up undefined/false and show invalid even if they had a value previously

**Review by nbintertech — CHANGES_REQUESTED** [review]

**Comment by ysintertech** [comment]
created separate issue for the whiteout styling bug #5536

File: src/app/waste-water/activated-sludge-form/activated-sludge-form.component.html [hunk 2]
```diff
@@ -26,10 +26,54 @@ <h3>Activated Sludge Data</h3>
             </span>
         </div>
         <!-- So -->
+
+        <div class="form-group" *ngIf="!this.form.controls.isUserDefinedSo.value"
+        [ngClass]="{'indicate-different': activatedSludgeDifferent.influentCBODBefore, 'invalid': form.controls.influentCBODBefore.invalid}">
+            <label>Influent CBOD<sub>5</sub> Before</label>
```
Comment (nbintertech) [inline src/app/waste-water/activated-sludge-form/activated-sludge-form.component.html:32]: The two new fields still don't use disabled styling (greyed out) when they aren't selected. Use the [readonly]="!selected" like the other fields on this form

**Review by nbintertech — CHANGES_REQUESTED** [review]

**Review by nbintertech — APPROVED** [review]

**Pull request merged**